osf_tests/test_files.py

```diff
@@ -51,3 +52,16 @@ def test_active_manager_does_not_return_trashed_file_nodes(project, create_test_
     assert BaseFileNode.objects.filter(node=project).count() == 3
     # root folder + file = 2 BaseFileNodes
     assert BaseFileNode.active.filter(node=project).count() == 2
+
+def test_folder_update_calls_folder_update_method(project, create_test_file, monkeypatch):
```

Comment:
This test is a bit strange--why not just test the behavior of Folder.update rather than monkeypatching it?

That said, this is not a blocker.